Vagrantfile

```diff
@@ -45,8 +45,10 @@ broadcast_address = n | (~n.instance_variable_get(:@mask_addr) & IPAddr::IN4MASK
 cluster_ip_cidr = settings["pod_network"]["cluster_ip_cidr"]
 service_ip_cidr = settings["pod_network"]["service_ip_cidr"]
 
+VAGRANT_PROVIDER = settings["conf"]["vagrant_provider"]
```

Comment:
we are now using lower case for the variables, so it should be "vagrant_provider"